Mathlib/LinearAlgebra/Matrix/GeneralLinearGroup.lean

```diff
@@ -329,4 +331,92 @@ def planeConformalMatrix {R} [Field R] (a b : R) (hab : a ^ 2 + b ^ 2 ≠ 0) :
 -/
 end Examples
 
+section cardinal
+
+variable (n : ℕ) {𝔽 : Type*} [Field 𝔽] [Fintype 𝔽]
+
+local notation "q" => Fintype.card 𝔽
+
+noncomputable instance {k : ℕ} :
```

Comment:
Those are the instance that create the problem in the other file, right? What about making them `lemma` and make them local instances in this section?

Reply:
Anyway you have several `{k : ℕ} {s : { s : Fin k → Fin n → 𝔽 // LinearIndependent 𝔽 s }}`. You can just put those in a `variable` line.